### PR TITLE
change max capacity

### PR DESCRIPTION
**Pull request by fafan**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/audio-conferencing-in-office-365.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ description: Learn about how Audio Conferencing in Microsoft 365 or Office 365 a
 ---
 
 # Audio Conferencing in Microsoft 365
-Audio Conferencing in Microsoft 365 and Office 365 enables users to call in to meetings from their phones. Audio Conferencing allows up to 250 phone attendees.
+Audio Conferencing in Microsoft 365 and Office 365 enables users to call in to meetings from their phones. Audio Conferencing allows up to 1000 phone attendees.
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-in-office-365.md:31]:
```suggestion
Audio Conferencing in Microsoft 365 and Office 365 enables users to call in to meetings from their phones. Audio Conferencing allows up to 1,000 phone attendees.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @fafan, kindly commit the minor suggested change. Many thanks.

**Comment by fafan** [comment]
> Copy review - @fafan, kindly commit the minor suggested change. Many thanks.

Thanks!

**Comment by fafan** [comment]
> Copy review - @fafan, kindly commit the minor suggested change. Many thanks.

when it will merge to live?

**Pull request merged**